wasp/boards/manifest_240x240.py

```diff
@@ -7,7 +7,6 @@
     'apps/calc.py',
     'apps/clock.py',
     'apps/chrono.py',
-    'apps/disaBLE.py',
```

Comment:
there are a bunch of other places where you need to remove references to DisaBLE:

```
$ git grep '[dD]isaBLE'
README.rst:.. image:: res/DisaBLEApp.png
docs/apps.rst:.. automodule:: apps.disaBLE
wasp/apps/disaBLE.py:"""DisaBLE
wasp/apps/disaBLE.py:.. figure:: res/DisaBLEApp.png
wasp/apps/disaBLE.py:class DisaBLEApp():
wasp/apps/disaBLE.py:    NAME = 'DisaBLE'
wasp/apps/disaBLE.py:    # 1-bit RLE, 96x64, generated from res/disaBLE_icon.png, 167 bytes
wasp/apps/software.py:        db.append(('disaBLE', factory('DisaBLE')))
wasp/boards/manifest_240x240.py:    'apps/disaBLE.py',
```

Also the `res/DisaBLEApp.png` and `res/disaBLE_icon.png` files

Reply:
Thanks!